apps/condo/lang/ru/ru.json

```
@@ -3307,7 +3307,7 @@
   "ticket.category.classifier.SURVEY.name": "Обследование",
```

Maybe smth like that?
```suggestion
  "ticket.category.classifier.INSPECTION.name": "Обследование",
```